src/core/index.ts

```diff
@@ -25,24 +27,13 @@ const getUserInfo = (context: Context): UserInfo => {
   return info;
 };
 
-const initPullRelatedRequest = (context: Context): Record<string, unknown> => {
-  const pullNumber = context.payload.pull_request.number;
-  const repo = context.payload.repository.name;
-  const owner = context.payload.repository.owner.login;
-  context.log.info(
-    `Initializing pull related request with ${owner}/${repo} #${pullNumber}`,
-  );
-  const req: Record<string, unknown> = { owner, repo };
-  req["pull_number"] = pullNumber;
-  return req;
-};
+type ReviewEvent = "APPROVE" | "REQUEST_CHANGES" | "COMMENT" | undefined;
 
 const approveChange = async (context: Context): Promise<void> => {
-  const req = (initPullRelatedRequest(
-    context,
-  ) as unknown) as Octokit.RequestOptions &
-    Octokit.PullsCreateReviewParamsDeprecatedNumber;
-  req.event = "APPROVE";
+  const req = context.repo({
+    "pull_number": context.payload.pull_request.number,
+    "event": "APPROVE" as ReviewEvent,
```

Comment:
Codacy found an issue: [The key 'event' is not sorted alphabetically](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6106171)